core/feature_flag_list.py

```diff
@@ -264,5 +269,19 @@ class FeatureNames(enum.Enum):
             'and makes it accessible to learners.',
             feature_flag_domain.ServerMode.DEV
         )
+    ),
+    FeatureNames.AUTOMATIC_VOICEOVER_REGENERATION_FROM_EXP.value: (
+        (
+            'The flag enables the automatic regeneration of voiceovers '
+            'directly from the exploration editor page.',
+            feature_flag_domain.ServerMode.DEV
+        )
+    ),
+    FeatureNames.LABEL_ACCENT_TO_VOICE_ARTIST.value: (
```

Comment:
The feature to label accents to voice artists is already present in the PROD, since we wanted to gate this functionality hence the feature flag is directly added in the PROD mode. PTAL at this issue: https://github.com/oppia/oppia/issues/20951.
Thanks

Reply:
Should this information be added to the PR description instead?

Reply:
Done